### ISSUE TITLE
How can I make sure 'lerna version' ignore certain packages?

### ISSUE THREAD

**Issue by dorsharon** [lerna/lerna#1586]
Is there any way I can commit changes to two packages, using conventional commits, so that lerna version (and therefore publish from-git) will ignore one of them?

## Expected Behavior
Make changes in pkgA and pkgB without wanting to release a new version of pkgB (let's say it's a small documentation change).
Commit those changes in a way that shows it only affects pkgA.
Lerna version and publish only affect pkgA.

## Current Behavior
I don't know of any way to do that.

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 3.1.1 |
| `npm --version`  | 5.8.0 |

-->

**Comment by dorsharon** [comment]
In my case, I'm managing a Storybook monorepo project with Lerna.
Now let's say someone had commited changes for two packages under the same commit, and mentioned BREAKING CHANGES that are relevant only to one of them.

The result would be that the 2nd package would get bumped major, which is unwanted.

In the past, cz-lerna-changelog could allow you to mention which package was actually affected, but since lerna v3 it no longer works.

**Comment by evocateur** [comment]
I don't maintain `cz-lerna-changelog`, so I can't speak to what "should" have happened. Lerna does not use conventional-changelog (via `--conventional-commits`) to determine _what_ changed, it only uses the conventional commits to suggest the version bump and compile a changelog.

To control the recommended version bump as you desire, you need to commit the documentation update seperately. That's not really a divergence from how `conventional-changelog` has always worked.

To control which diffs are recognized by lerna as a "changed" package, use the [`--ignore-changes` option](https://github.com/lerna/lerna/tree/master/commands/version#--ignore-changes). Note that lerna will always bump dependents who have dependencies that have changed, so you still need to separate the commits if you don't want the "BREAKING CHANGE" to "bleed" into the dependent package version bump.

**Comment by NemoOudeis** [comment]
@evocateur thanks for clarifying that behavior. 

I have a followup question, with the following (very similar) background. Say i have 2 packages like this:

```
packages
├── apple
└── banana
```

I made code changes in `apple` and I removed a dependency from `banana`'s `package.json` (because it was in there by accident).

Now I want to release only `apple`, because `banana` didn't really change. I do
```
lerna version --ignore-changes 'packages/banana/**'
``` 
So my intuition would be that this will bump only `apple`'s version and `banana` would be ignored.

Your statement 

> Note that lerna will always bump dependents who have dependencies that have changed

sounds to me like lerna will force a version bump in `banana` regardless though, even if explicitly ignored.

Is my understanding correct? Is that intended behavior?
